astropy/io/ascii/tests/test_c_reader.py

```diff
@@ -438,9 +438,13 @@ def test_doubled_quotes(read_csv):
 
     """
     # fmt: off
-    tbl = '\n'.join(['a,b',
-                     '"d""","d""q"',
-                     '"""q",""""'])
+    tbl = dedent(
```

Comment:
Here, I'd blackify the lines but just keep the quoting.

Reply:
Done.